src/data/markdown/translated-guides/en/07 Testing Guides/05 Running distributed tests.md

```diff
@@ -0,0 +1,379 @@
+</Blockquote>
+
+Let's go ahead verify our script is valid by performing a brief test:
+```shell
+k6 run test.js
+```
+
+We should see a successful execution and resulting output summary.
+
+## 3. Add test scripts
+In order for your test scripts to be run, they must be published into your cluster.
+This can be done using [ConfigMap](https://kubernetes.io/docs/concepts/configuration/configmap/) or [PersistentVolume](https://kubernetes.io/docs/concepts/storage/persistent-volumes/) resources.
+
+### Add as a ConfigMap
+Using a `ConfigMap` is a quick and straightforward mechanism for adding your test scripts to Kubernetes.
+The `kubectl` tool provides a convenient method to create a new `ConfigMap` from a local script.
+
+Let's create our ConfigMap as `my-test` with the content of our `test.js` script we created in the previous step:
+
+```shell
+kubectl create configmap my-test --from-file test.js
+```
+
+<Blockquote mod="attention" title="">
+
+Limitations exist on how large your test script can be when deployed within a `ConfigMap`.
+Kubernetes imposes a size limit of 1,048,576 bytes (1 MiB) for the data, therefore if your test scripts exceed this limit, you'll need to mount a `PersistentVolume`.
+
+Check the [motivations](https://kubernetes.io/docs/concepts/configuration/configmap/#motivation) for when you should use a `ConfigMap` versus a `PersistentVolume`.
+
+</Blockquote>
+
+You should see confirmation with `configmap/my-test created`.
+
+### Add inside a PersistentVolume
+Setting up a `PersistentVolume` is beyond the scope of this guide, but enables access to a shared filesystem from your Kubernetes cluster.
+
+When using this option, organize your test scripts in the applicable filesystem just as you would locally.
+This mechanism is ideal when breaking up monolithic scripts into reusable [modules](/using-k6/modules/).
+
+<Blockquote mod="note" title="As seen on k6 Office Hours">
+
+Organizing your test scripts was part of the discussion during [episode #76](https://www.youtube.com/watch?v=zDtEzp_JUOE&utm=k6-guides) of k6 Office Hours.
+
+</Blockquote>
+
+<Blockquote mod="attention" title="">
+
+When using a `PersistentVolume`, the operator will expect all test scripts to be contained within a directory named `/test/`.
+
+</Blockquote>
+
+To learn more about creating `PersistentVolume` resources, review the [Kubernetes documentation](https://kubernetes.io/docs/concepts/storage/persistent-volumes/).
+
+## 4. Create a custom resource
+During [installation](#install-the-operator), the `K6` [Custom Resource](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/) was added to the Kubernetes API.
+The data we provide in the custom resource contains all the information necessary for the k6 operator to start a distributed load test.
+
+The main elements defined within the `K6` resource relate to the name and location of the test script to run, and the amount of [parallelism](/misc/glossary/#parallelism) to utilize.
```

Comment:
```suggestion
During [installation](#install-the-operator), the `K6` [Custom Resource definition](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/) was added to the Kubernetes API.
The data we provide in the custom resource `K6` object should contain all the information necessary for the k6-operator to start a distributed load test.

Specifically, the main elements defined within the `K6` object relate to the name and location of the test script to run, and the amount of [parallelism](/misc/glossary/#parallelism) to utilize.
```
Explanation: here, I thought it'd be nice to clarify what is being installed automatically during "installation of operator" step VS what a user is supposed to define themselves. Otherwise, it sounds as if user is supposed to define CRD itself - which is not true.

Reply:
> It seems you've ended up adding the k8s link in the next paragraph 😄

😄 yes...I continued to work with it after the initial reply.